enum.h

```diff
@@ -700,6 +717,15 @@ Enum::_from_value_loop(Enum::_integral value, std::size_t index)               \
                 _from_value_loop(value, index + 1);                            \
 }                                                                              \
                                                                                \
+BETTER_ENUMS_CONSTEXPR_ inline Enum::_optional                                 \
+Enum::_from_index_loop(std::size_t index)                                      \
```

Comment:
I recommend not calling this `..._loop`, because it doesn't loop over the indexes, but just does a direct lookup. In fact, it looks like just the implementation of `_from_index_nothrow`, so I suggest moving it there.